### DISCUSSION TITLE
menyoo crashes when pausing or changing time how do i fix this?

### DISCUSSION THREAD

**Discussion by JesperYouTube** [MAFINS/MenyooSP#618] [Q&A]
can someone help me, i tried to change time and then a core error appeared, how do i fix this, it is getting annoying and yes i updated scripthookv and dot net and yes i tried re-installing it, there the answears of those regular questions are gone now i just need a fix for this

**Comment by Fidelitygang** [comment]
same problem

**Comment by AlphaPrime108** [comment]
same here
